pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1217,6 +1217,76 @@ def test_setitem_array_as_cell_value(self):
         expected = DataFrame({"a": [np.zeros((2,))], "b": [np.zeros((2, 2))]})
         tm.assert_frame_equal(df, expected)
 
+    def test_iloc_setitem_nullable_2d_values(self, using_array_manager, request):
+        if using_array_manager:
+            mark = pytest.mark.xfail(reason="Goes through split path, loses dtype")
```

Comment:
Can you use the predefined array_manager_not_yet_implemented mark instead (better searchable)?

Reply:
I already prepared that :) -> https://github.com/pandas-dev/pandas/pull/44571